src/slam_toolbox.cpp

```diff
@@ -1211,6 +1215,12 @@ int main(int argc, char** argv)
 /*****************************************************************************/
 {
   ros::init(argc, argv, "slam_toolbox");
+  ros::NodeHandle nh;
+  const rlim_t max_stack_size = 16777216;
```

Comment:
... what's this magic number? can't we round to something nice, say 20MB

Reply:
and add as a top preprocessor variable. `#define max_stack_size 20000000 //comment explaining`